README.md

```diff
@@ -26,7 +26,7 @@ The CLO (Cluster Logging Operator) *owns* the following APIs:
 
 To install a released version of cluster logging see the [Openshift Documentation](https://docs.openshift.com/)
 
-To experiment or contribute to the development of cluster logging, see [HACKING.md](./docs/HACKING.md)
+To experiment or contribute to the development of cluster logging, see [HACKING.md](./HACKING.md)
```

Comment:
Is there a need to move the doc?  more discoverable?

Reply:
I think it's more conventional - its the pattern in other openshift repos.

Reply:
I've seen it in other places too - README for users HACKING for contributors.

Reply:
All this stuff taken out of this PR, will do a separate PR to improve the HACKING doc.